nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "watch": ["api/", "api-lib/", "scripts/"],
+  "delay": 3000,
```

Comment:
Why 3000? Seems a bit long. I think of the delay being useful if you are saving multiple files, but it seems long if I'm going to "edit code", "save", "reload" and need to wait for the reload to be processed?

Reply:
prettier takes more than a second and this debounces any file changes due to orettier